TESTING_GUIDE.md

```diff
@@ -29,6 +29,18 @@ We have devised our own test framework to test the spec against an OpenSearch cl
 
 ### Running Spec Tests Locally
 
+Prerequisites: 
+Download and install latest version of Node.js and npm from 
+[https://docs.npmjs.com/downloading-and-installing-node-js-and-npm]
+```bash
+Note: to download the latest version of npm, on the command line, run the following command:
+npm install -g npm
+```
+
+In Windows using PowerShell:
```

Comment:
Since tests don't work on Windows, remove the part about Windows for now. Once you've fixed tests to run on Windows we can add this.